core/tests/webdriverio_desktop/contributorAdminDashboard.js

```diff
@@ -286,12 +286,12 @@ describe('Contributor Admin Dashboard', function() {
 
     await contributorDashboardAdminPage.navigateToQuestionSubmitterTab();
     await contributorDashboardAdminPage.waitForLoadingMessageToDisappear();
-    await contributorDashboardAdminPage.expectStatsElementCountToBe(2);
+    await contributorDashboardAdminPage.expectStatsElementCountToBe(1);
```

Comment:
Just to check -- why did this change?

Reply:
This change is because the stats component was being drawn twice on the page, once for mobile and once for desktop. I changed it so that it  is being drawn once, either mobile or desktop, I don't think it breaks anything, I tried asking Kshitij for some input but haven't heard back yet. The change is one of a number I made to reduce the number of times that the stats component was being drawn and the associated api call being made  ( I amalgamated the variables passed in to the stats component into 1 variable, added the topicsAreFetched variable, made the filter only change when something on it had changed). It was being drawn 8 times, its down to 2 now, I don't think it can go any lower due to the angular js component.

Reply:
Thanks -- LGTM. Just one concern here ... it sounds like this is set up to count hidden elements that the user doesn't see (since if they are on mobile they won't see desktop, and vice versa).

That seems wrong, shouldn't the test be structured to only detect stuff that is actually visible to the user?

Reply:
I've already added this *ngIf="checkMobileView() in the contributor-admin-dashboard.component.html page, so that only the mobile view is shown, and hence the hidden element is no longer present. There was already a check on *ngIf="!checkMobileView() for the desktop view. 

I was a little concerned because the e2e test ( the one you have highlighted ) specifically checked to see that 2 components were present, even though only 1 is visible. But I did some manual testing around this and i can't see how anything is broken.

Reply:
Hm, ok, that's weird, I wonder whether the test is actually detecting the right thing?

Though if you feel it's all good, no concerns about merging. Thanks!